old-README.md

```diff
@@ -0,0 +1,44 @@
+## CARO
```

Comment:
Why check that in? Combine the two READMEs into one.

Reply:
looked at it and didnt think it was relevant to the new format but didnt want to delete it - I'll take a look again and I'll move this to old files instead.